packages/squiggle-lang/__tests__/Distributions/KlDivergence_test.res

```diff
@@ -60,9 +60,52 @@ describe("kl divergence", () => {
   })
 })
 
-describe("combine along support test", () => {
+describe("klDivergence: discrete -> discrete -> float", () => {
+  let klDivergence = DistributionOperation.Constructors.klDivergence(~env)
+  let mixture = a => DistributionTypes.DistributionOperation.Mixture(a)
+  exception KlFailed
+  exception MixtureFailed
+  let float1 = 1.0
+  let float2 = 2.0
+  let float3 = 3.0
+  let point1 = mkDelta(float1)
+  let point2 = mkDelta(float2)
+  let point3 = mkDelta(float3)
+  let a' = [(point1, 1e0), (point2, 1e0)]->mixture->run
+  let b' = [(point1, 1e0), (point2, 1e0), (point3, 1e0)]->mixture->run
+  let (a, b) = switch (a', b') {
+  | (Dist(a''), Dist(b'')) => (a'', b'')
+  | _ => raise(MixtureFailed)
+  }
```

Comment:
This looks like it should be in a fixtures file

Reply:
will do